# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Define goal
float32[] goal_pos
float32[] goal_vel
uint32 rate
float32 time
